src/cmd-buildprep

```diff
@@ -3,118 +3,95 @@
 #
 # Fetches the bare minimum from external servers to create the next build.
```

Comment:
if connecting to s3 does a user need to have credentials set ? If so maybe we should mention that at the top here